### PR TITLE
doc: include FAR in ThirdPartyToolProfiles.md

### PR DESCRIPTION
**Pull request by vtabota**
## Summary of the Pull Request

New 3rd part tool "Far Manager" configuration.

 
## PR Checklist
* [ ] Closes #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

File: doc/user-docs/ThirdPartyToolProfiles.md [hunk 1]
```diff
@@ -51,6 +51,17 @@ Note that the starting directory of Cygwin is set as it is to make the path
 work. The default directory opened when starting Cygwin will be `$HOME` because
 of the `--login` flag.
 
+## Far Manager
+
+```json
+{
+    "name" : "Far",
+    "commandline" : "cmd.exe /c \"c:\\program files\\far manager\\far.exe\"",
```
Comment (miniksa) [inline doc/user-docs/ThirdPartyToolProfiles.md]: Can't just launch straight into far.exe?

Comment (DHowett-MSFT) [reply doc/user-docs/ThirdPartyToolProfiles.md]: Yeah, wut

Comment (vtabota) [reply doc/user-docs/ThirdPartyToolProfiles.md]: Actually, yes it can.

**Comment by DHowett-MSFT** [comment]
Thanks!

**Pull request merged**